From d56759aaff209204957b25a5449de33efb7718ea Mon Sep 17 00:00:00 2001
From: Dmitry Shachnev <mitya57@gmail.com>
Date: Thu, 25 Aug 2016 21:11:37 +0300
Subject: Reverted the conversion of bytes to 'const char *'

The latter is not supported by QMetaType and gives QSignalSpy a problem.

Patch-Name: bytes_conversion.diff
---
 qpy/QtCore/qpycore_chimera.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/qpy/QtCore/qpycore_chimera.cpp b/qpy/QtCore/qpycore_chimera.cpp
index 920fee6..2c8f8b8 100644
--- a/qpy/QtCore/qpycore_chimera.cpp
+++ b/qpy/QtCore/qpycore_chimera.cpp
@@ -416,11 +416,6 @@ bool Chimera::parse_py_type(PyTypeObject *type_obj)
         _type = sipType_QString;
         _metatype = QMetaType::QString;
     }
-    else if (type_obj == &PyBytes_Type)
-    {
-        _metatype = -1;
-        _name = "const char*";
-    }
 #else
     else if (type_obj == &PyString_Type || type_obj == &PyUnicode_Type)
     {
